spec.html

```diff
@@ -473,10 +489,10 @@ <h1>Tree Grammar</h1>
 interface ExportFromSpecifier : Node {
   // The only `IdentifierName in `ExportSpecifier :: IdentifierName`,
   // or the first in `ExportSpecifier :: IdentifierName as IdentifierName`.
-  attribute IdentifierName name;
+  attribute IdentifierReference name;
   // The second `IdentifierName` in `ExportSpecifier :: IdentifierName as IdentifierName`,
   // if that is the production represented.
-  attribute IdentifierName? exportedName;
+  attribute PropertyName? exportedName;
```

Comment:
This should be same type as `ExportLocalSpecifier.exportedName` (see the comment for it)